waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -308,6 +308,12 @@ proc findMessages(w: WakuStore, query: HistoryQuery): HistoryResponse =
   result = HistoryResponse(messages: newSeq[WakuMessage]())
   # data holds IndexedWakuMessage whose topics match the query
   var data = w.messages.filterIt(it.msg.contentTopic in query.topics)  
+
+  # temporal filtering
+  if (query.endTime > query.startTime):
```

Comment:
Since the endpoints are inclusive in the tests below, I think `(query.endTime >= query.startTime)` may be the more general condition (this relates to my comments on the test).

Reply:
Sure, makes sense!